sunpy/net/helio/hec.py

```diff
@@ -85,8 +86,8 @@ def __init__(self, link=None):
             # The default wsdl file
             link = parser.wsdl_retriever()
         session = Session()
-        # Disable SSL check.
-        session.verify = False
+        # This is for use in our test suite.
+        session.verify = int(os.environ.get("HELIO_SSL", 1))
```

Comment:
I'm being very thick here but it works.

Reply:
If I passed a bool in the tox env, it came back as "True" or "False" which is always true. 
So int a 0 or 1 gets me the boolean I need. But it seems a bit silly.

Reply:
All we really need to be testing for is if this var is set at all, the value is somewhat irrelevant.
```suggestion
        session.verify = bool(os.environ.get("VERIFY_HELIO_SSL", 1))
```